src/js/otus/types.ts

```diff
@@ -84,7 +84,7 @@ export type OTUMinimal = {
 /** A complete OTU */
 export type OTU = OTUMinimal & {
     isolates: Array<OTUIsolate>;
-    issues?: { [key: string]: any } | boolean;
```

Comment:
Sadly the old typing is correct and the UI needs to be able to handle it atm:

https://github.com/virtool/virtool-core/blob/d4acce2e9c1fe5c6b9421854702bb1911b746d1a/virtool_core/models/otu.py